app/javascript/pages/country/widget/widgets/widget-fao-forest/widget-fao-forest-component.js

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import Loader from 'components/loader/loader';
+import WidgetHeader from 'pages/country/widget/components/widget-header';
+import WidgetPieChart from 'pages/country/widget/components/widget-pie-chart';
+import WidgetPieChartLegend from 'pages/country/widget/components/widget-pie-chart-legend';
+import './widget-fao-forest-styles.scss';
+
+class WidgetFAOForestGain extends PureComponent {
+  render() {
+    const { locationNames, isLoading, data } = this.props;
+    const chartData = data.chartData ? data.chartData : [];
```

Comment:
Can this not be handled in the container? Or in the data={} prop of the pie chart?

Reply:
Or just dont render Pie chart until `data &&`